### ISSUE TITLE
Aerexplorer 24H2 compatability issue?

### ISSUE THREAD

**Issue by explor202** [ramensoftware/windhawk-mods#1203]
I've installed the Aerexplorer mod on 2 different 23H2 PC's, and it works great. I specifically use it to remove the suffix "file explorer" from displaying for each folder I have open. However, I'm NOT able to get that to work on a 24H2 PC. @aubymori  is that a comparability issue?

**Comment by aubymori** [comment]
> I've installed the Aerexplorer mod on 2 different 23H2 PC's, and it works great. I specifically use it to remove the suffix "file explorer" from displaying for each folder I have open. However, I'm NOT able to get that to work on a 24H2 PC. @aubymori is there a comparability issue with 24H2?

Yes. Aerexplorer was designed for Windows 10 2004-22H2, not Windows 11. I'm surprised you even got it working on 23H2.

**Comment by yw4z** [comment]
@aubymori do you have any plans to make it compatible with win11? or making a separate version

**Comment by aubymori** [comment]
> @aubymori do you have any plans to make it compatible with win11? or making a separate version

At the moment, there are no plans for Aerexplorer to support Windows 11.

**Comment by ManGuyNY** [comment]
> > I've installed the Aerexplorer mod on 2 different 23H2 PC's, and it works great. I specifically use it to remove the suffix "file explorer" from displaying for each folder I have open. However, I'm NOT able to get that to work on a 24H2 PC. @aubymori is there a comparability issue with 24H2?
> 
> Yes. Aerexplorer was designed for Windows 10 2004-22H2, not Windows 11. I'm surprised you even got it working on 23H2.

Can confirm it somehow runs for me as well on Win 11 23H2. Though it does cause explorer.exe to crash periodically...I love it too much to disable it. T_T

**Comment by yw4z** [comment]
btw OldNewExplorer still works with windows 11 but i got few crashes and caused some apps to crash while trying to open file with file explorer popup, so injection still works but there is some issues. probably can be fixed with minor changes. i have no luck with aerexplorer but "Windows 7 Command Bar" and "Remove command Bar" works. I got this problems with them; menu bar appears sometimes and titlebar not visible on first launch

**Comment by explor202** [comment]
I've rolled back all 8 of my PC's to 22H2 and installed WinSetView where needed.

**Comment by m417z** [comment]
> I specifically use it to remove the suffix "file explorer" from displaying for each folder I have open.

I just created a new mod specifically for that: [Remove File Explorer Suffixes](https://windhawk.net/mods/file-explorer-remove-suffixes).

**Comment by yw4z** [comment]
@m417z i really need one that can use windows 7 file explorer without command bar but none of mods working on 24H2 currently

**Comment by m417z** [comment]
@yw4z sure, my comment doesn't imply that this issue is solved, I just commented about this one specific use case which now has an alternative solution.

**Comment by yw4z** [comment]
yes i know, thanks for your reply

**Comment by albino1** [comment]
> I just created a new mod specifically for that: [Remove File Explorer Suffixes](https://windhawk.net/mods/file-explorer-remove-suffixes).

Amazing, works like a charm, thanks!
